lib/search_solr_tools/harvesters/tdar.rb

```diff
@@ -14,14 +14,22 @@ def harvest_and_delete
       end
 
       def harvest_tdar_into_solr
-        start_record = 1
+        start_record = 0
+        total_harvested = 0
+        total_expected = total_results
```

Comment:
This was initially confusing. Is it un-ruby to put a `()` after `total_results`? e.g., `total_expected = total_results()`